released/packages/coq-compcert/coq-compcert.3.7~platform-flocq/files/copy_ini_to_coqlib.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/Makefile b/Makefile
+index af069e3f..c924283b 100644
+--- a/Makefile
++++ b/Makefile
+@@ -205,18 +205,19 @@ compcert.ini: Makefile.config
+          echo "prepro=$(CPREPRO)"; \
+          echo "linker=$(CLINKER)"; \
+          echo "asm=$(CASM)"; \
+-	 echo "prepro_options=$(CPREPRO_OPTIONS)";\
+-	 echo "asm_options=$(CASM_OPTIONS)";\
+-	 echo "linker_options=$(CLINKER_OPTIONS)";\
++         echo "prepro_options=$(CPREPRO_OPTIONS)";\
++         echo "asm_options=$(CASM_OPTIONS)";\
++         echo "linker_options=$(CLINKER_OPTIONS)";\
+          echo "arch=$(ARCH)"; \
+          echo "model=$(MODEL)"; \
+          echo "abi=$(ABI)"; \
+          echo "endianness=$(ENDIANNESS)"; \
++         echo "bitsize=$(BITSIZE)"; \
+          echo "system=$(SYSTEM)"; \
+          echo "has_runtime_lib=$(HAS_RUNTIME_LIB)"; \
+          echo "has_standard_headers=$(HAS_STANDARD_HEADERS)"; \
+          echo "asm_supports_cfi=$(ASM_SUPPORTS_CFI)"; \
+-	 echo "response_file_style=$(RESPONSEFILE)";) \
++         echo "response_file_style=$(RESPONSEFILE)";) \
```

Comment:
This seems completely unrelated to installing compcert.ini at the right place.

Reply:
Ehm yes, I also added a bitsize field to the file afterwards, since it was missing and VST needs it - and fixed the spacing on the way (it was a mix of tabs and spaces from one line to another).

Reply:
Same remark as bellow : if the patch (or PR https://github.com/AbsInt/CompCert/pull/348) is not only about installing the .ini file, then document it accordingly.